### PR TITLE
feat(query): added AKS Disk Encryption Set ID Undefined

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if disk_encryption_set_id is not being used in AKS

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/azure/aks_uses_disk_encryption_set/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "b17d8bb8-4c08-4785-867e-cb9e62a622aa",
+  "queryName": "AKS Uses Disk Encryption Set",
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/aks_uses_disk_encryption_set/metadata.json:3]: The query name could be more intuitive like "AKS Disk Encryption Set ID Undefined" (just a suggestion)

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/azure/aks_uses_disk_encryption_set/metadata.json (hunk 1 above)
Comment (rogeriopeixotocx) [reply assets/queries/terraform/azure/aks_uses_disk_encryption_set/metadata.json:3]: I agree. Don't forget to change the directory name too.

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]

**Pull request merged**